### ISSUE TITLE
Int object has no attribute: getitem

### ISSUE THREAD

**Issue by iceanfire** [panalysis/Google2Pandas#2]
Running into the followoing error whilst following the demo example:

Traceback (most recent call last):
  File "conversion_rate.py", line 26, in <module>
    df, formatted_qry = ga.execute_query(**query)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/google2pandas/_panalysis_ga.py", line 195, in execute_query
    raise ValueError('Error making query: {0}'.format(e))
ValueError: Error making query: 'int' object has no attribute '__getitem__'

**Comment by DeliciousHair** [comment]
That's odd, can you post your query(without any ids field) please?

Also, I think I left the ids field needing to be passed as stirng-like:

```
query = {'ids' : '12345', ...}
```

**Comment by iceanfire** [comment]
```
query = {'ids': 12345678,
         'metrics': 'pageviews',
         'dimensions': ['date', 'pagePath', 'browser'],
         'filters': ['pagePath=~iPhone', 'and', 'browser=~Firefox'],
         'start_date': '8daysAgo',
         'max_results': 10
}
```

(put a dummy id in there)

**Comment by DeliciousHair** [comment]
Yeah, put it in there as a string, that should fix it for now. That behaviour should be fixed though.

**Comment by iceanfire** [comment]
Awesome, thanks!

**Comment by DeliciousHair** [comment]
Feel free to fix that, it should accept either

**Comment by DeliciousHair** [comment]
fixed with #3
